first_responder.md

```diff
@@ -0,0 +1,121 @@
+# Guidelines for first responder
+
+The DLSS infrastructure team is trying out a new rotating role of "first responder".  PRs are always welcome, but especially while we're ironing things out and having regular retrospectives about the process.  If it sticks, and/or the access team adopts it, we can get rid of this caveat.
+
+## The "first responder" rotation premises
+
+* Respond to production issues in a timely fashion during business hours.
+* Single process for handling questions outside the team's current work cycle.
+* Share the responsibility for production outages across the team in a planned way.
+* Every team member rotates through this responsibility.
+* Encourage cross-training, since a first-responder will likely have to investigate applications with which they are unfamiliar.
+* Shore up missing and outdated documentation of production code and processes for everyone (ops, devops, PSM, stakeholders, devs, etc), informed by actual attempts to find and use said documentation to investigate production issues.
+
+### "First responder" != "on call"
+
+* The idiosyncratic name of the role is intentional.  It is not the duty of the "first responder" to be on-call after hours.  The first responder rotation is an effort to watch and triage production issues in an intentional and organized fashion, since no engineer was officially assigned this responsibility in the past, and such minding of things was haphazard.
```

Comment:
Should indicate that this is during a first responders normal working hours, i.e., a first responder in a different time zone may monitor during Stanford business hours, but does not have an extraordinary requirement while on duty.

Reply:
fixed, thanks for the suggestion!